frontend/assets/l10n/main/app_en.json

```
@@ -90,7 +94,18 @@
     "flashOff": "Flash off",
     "flashOn": "Flash on",
     "internetRequired": "An internet connection is required.",
+    "moreActionsActivateDescription": "Your saved Ehrenamtskarte will be retained. You can remove it manually.",
```

```suggestion
    "moreActionsActivateDescription": "Your saved Ehrenamtskarten will be retained. You can remove them manually.",
```

---

I think I will go for (same below):
```suggestion
    "moreActionsActivateDescription": "Your saved cards will be retained. You can remove them manually.",
```